gc/base/IndexableObjectScanner.hpp

```diff
@@ -19,7 +19,7 @@
 #if !defined(INDEXABLEOBJECTSCANNER_HPP_)
 #define INDEXABLEOBJECTSCANNER_HPP_
 
-#include "ObjectScanner.hpp"
+#include "../ObjectScanner.hpp"
```

Comment:
These are being inserted by configure. I did not add relative path (avoid them altogether if I can).

Will remove and look for others. Should probably open an issue relating to relative paths and configure. This was Robert's ides if I recall. If he backed it out he may have missed something.